src/common/Datastore.ts

```diff
@@ -133,24 +133,57 @@ export default class Datastore {
   private static setupReplicationRequestSchema () {
     const replicationRequestSchema = new Schema<ReplicationRequest>({
       datasetId: Schema.Types.String,
-      minReplicas: Schema.Types.Number,
-      client: Schema.Types.String,
+      maxReplicas: Schema.Types.Number,
       criteria: Schema.Types.String,
```

Comment:
let's rename this to providers. you can also use string array

Reply:
But what about pando and other potential stuff? I thought this is an expression that can be anything.

Reply:
Not sure about that either. It's likely not going to happen. Let's then keep it this way then.